src/views/Pages/Page404/Page404.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { Button, Col, Container, Input, InputGroup, InputGroupAddon, InputGroupText, Row } from 'reactstrap';
+
+class Page404 extends Component {
+  render() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected 'this' to be used by class method 'render'.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)